.gitignore

```diff
@@ -1,4 +1,4 @@
 .vscode/
 dotm.code-workspace
 .DS_Store
-extracted/
+./extracted/
```

Comment:
did this work?  i don't see a long new list of files being added here... but i also don't think you erased the old ones.  it took me like a half hour last night to go in and erase one-by-one the files in github.  would have been a lot faster to do it on the computer directly and git push the changes :woman_facepalming: